.github/workflows/regression_test.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       matrix:
         rust:
           - stable
-          - 1.56
+          - 1.60
```

Comment:
the `toolchain` github action interpret it as 1.6.0, so it fail
can you fix it?

Reply:
I imagine [this](https://github.com/gftea/amqprs/pull/141/commits/ee402db6046fc227f72434555d20ddc5066c9ea8) should fix it.